python/lolopy/learners.py

```diff
@@ -260,10 +260,13 @@ def __init__(self, num_trees=-1, use_jackknife=True, bias_learner=None,
             use_jackknife (bool): Whether to use jackknife based variance estimates
             bias_learner (BaseLoloLearner): Algorithm used to model bias (default: no model)
             leaf_learner (BaseLoloLearner): Learner used at each leaf of the random forest (default: GuessTheMean)
-            subset_strategy (Union[string,int,float]): Strategy used to determine number of features used at each split in decision
-                trees. Can be "sqrt" for the square root of the number of features, an int to specify the number
-                of features, or a float to set the fraction of the total feature count. "auto" uses "sqrt" for
-                classification problems and 1/3 for regression problems.
+            subset_strategy (Union[string,int,float]): Strategy used to determine number of features used at each split 
+                Available options:
+                    "auto": Use the default for lolo (all features)
```

Comment:
"all features" is the default for regression, but not classification (which is sqrt, consistent with sklearn).

Reply:
Oops, I misread the Scala code